### ISSUE TITLE
Require First Name, Last Name for new user on Sign Up

### ISSUE THREAD

**Issue by joemcl** [ProgressiveCoders/Spoke#1113]
Our log in page now points to spokewf.auth0.com and the new Sign Up link does not require a first or last name, which obviously we need for each new user.

**Comment by cgalbiati** [comment]
I'm going to look into this a bit, but if someone else finds it first, let me know

**Comment by cgalbiati** [comment]
Turns out you can add validation to the custom login form.  I added it to the docs in this PR: https://github.com/MoveOnOrg/Spoke/pull/1115, but someone will have to make the update in the prod auth0 account as well

**Comment by joemcl** [comment]
@cgalbiati - the fix is this, per @bchrobot - ```Have you done step 11 (new-ish) in the readme?
https://github.com/MoveOnOrg/Spoke/blob/main/README.md ``` . 

We're back up and running with our  previous login page ( see 'Sign Up' at spoke.workingfamilies.org ).

But the thing is - we had the customized log in set up previously - since our initial deploy of Spoke on Heroku over a year ago. 

I copied and pasted the code in the link in step 11, above - and re-enabled the Customize Login Page toggle in auth0 settings. 

What changed from previously and I think is the cause of the change is that we moved from the Heroku 16 stack to the Container stack on Heroku. 

So @jlegrone - something about changing from Heroku 16 to Container stack and deploying upstream stage-main appears to disable Customize Login Page toggle in auth0 and remove previous customized code in that.  

cc: @deasterdaywfp .

**Comment by joemcl** [comment]
@bchrobot when I changed from Heroku 16 to Container and deployed stage-main in the container, that disabled the previous auth0 Customize Login Page toggle, with presumably the original Spoke Auth0 Lock code we used when setting up Spoke originally early last year. I turned on the Customize Login Page toggle and used the newer code from step 11 in the readme. 

Some users logged in or signed up as new during the window when the standard credentials were in place, which did not require first name or last name, so I'm adding those first and last to the user records in Spoke. 

Four of the folks who logged in during the window were already Spoke users, but, they used Google to auth, and now we have two user records for them, with same e-mail, but with auth0 ID in `auth0_id` in one user record, and google-oauth2 ID in the other. Which is not a big deal, I'll leave the two user records as they are after updating the newer record with first and last names and cells.

So @jlegrone just need to update the docker container instructions for folks/orgs like me and WFP, who move an existing Spoke instance on Heroku from Heroku 16 to Container, to re-enable Customize Log In toggle, and make sure using the latest code for that.
